scripts/deploy-helm.sh

```diff
@@ -272,7 +278,10 @@ if [[ $DEPLOYMENT_TYPE == all ]] || [[ $DEPLOYMENT_TYPE == logs ]]; then
   else
     echo "Using an existing Active Gate"
     check_if_parameter_is_empty "$DYNATRACE_LOG_INGEST_URL" "DYNATRACE_LOG_INGEST_URL"
-    check_url "$DYNATRACE_LOG_INGEST_URL" "$ACTIVE_GATE_TARGET_URL_REGEX" "Not correct dynatraceLogIngestUrl. Example of proper ActiveGate endpoint used to ingest logs to Dynatrace: https://<your_activegate_IP_or_hostname>:9999/e/<your_environment_ID>"
+    check_url "$DYNATRACE_LOG_INGEST_URL" "$DYNATRACE_URL_REGEX" "$ACTIVE_GATE_TARGET_URL_REGEX" \
+      "Not correct dynatraceLogIngestUrl. Example of proper ActiveGate endpoint used to ingest logs to Dynatrace:\n
+        - for Public ActiveGate: https://<your_environment_ID>.live.dynatrace.com\n
```

Comment:
change message - same as in comment above

Reply:
our DYNATRACE_URL_REGEX is already supporting managed (we already have one customer), provbide example in message

Reply:
I'm not sure we should mention it explicitly in docs or script help messages if it's in preview phase. Or would you like to have a disclaimer like:
```
- for direct ingest through the SaaS Cluster API: https://<your_environment_ID>.live.dynatrace.com\n
- for direct ingest through the Managed Cluster API (preview, only selected customers): https://<your_environment_ID>.live.dynatrace.com\n
```

Reply:
You are right, this is a preview, please disregard my previous comment about managed

Reply:
Alright, adjusted the message. Please re-review